### DISCUSSION TITLE
How to run admin app in locally

### DISCUSSION THREAD

**Discussion by ClarkDuck** [mozilla/hubs#4966] [Hubs Cloud Questions] [answered]
Hi,

I am a new man in hubs, How can I to run the admin app in locally?
I tried to `npm run start` in the `admin` directory. but the `https://hubs.local:8989` does not open the admin page.
Anybody knows the link of doc please help me.
Thanks in advance!

**Comment by pattersonbl2** [comment, marked as answer]
This would require a few configuration changes we don't have documented fully. I am aware of people in the discord that have set it up. This question would be great to also ask in our community discord. You can join here (https://discord.com/invite/dFJncWwHun). I will ask my team if they may have a quick answer as well.
